javalanglib/src/main/scala/java/lang/System.scala

```diff
@@ -130,55 +131,65 @@ object System {
 
     if (src == null || dest == null) {
       throw new NullPointerException()
-    } else (src match {
-      case src: Array[AnyRef] =>
-        dest match {
-          case dest: Array[AnyRef] => impl(src.length, dest.length, (i, j) => dest(i) = src(j))
-          case _                   => mismatch()
-        }
-      case src: Array[Boolean] =>
-        dest match {
-          case dest: Array[Boolean] => impl(src.length, dest.length, (i, j) => dest(i) = src(j))
-          case _                    => mismatch()
-        }
-      case src: Array[Char] =>
-        dest match {
-          case dest: Array[Char] => impl(src.length, dest.length, (i, j) => dest(i) = src(j))
-          case _                 => mismatch()
-        }
-      case src: Array[Byte] =>
-        dest match {
-          case dest: Array[Byte] => impl(src.length, dest.length, (i, j) => dest(i) = src(j))
-          case _                 => mismatch()
-        }
-      case src: Array[Short] =>
-        dest match {
-          case dest: Array[Short] => impl(src.length, dest.length, (i, j) => dest(i) = src(j))
-          case _                  => mismatch()
-        }
-      case src: Array[Int] =>
-        dest match {
-          case dest: Array[Int] => impl(src.length, dest.length, (i, j) => dest(i) = src(j))
-          case _                => mismatch()
-        }
-      case src: Array[Long] =>
-        dest match {
-          case dest: Array[Long] => impl(src.length, dest.length, (i, j) => dest(i) = src(j))
-          case _                 => mismatch()
-        }
-      case src: Array[Float] =>
-        dest match {
-          case dest: Array[Float] => impl(src.length, dest.length, (i, j) => dest(i) = src(j))
-          case _                  => mismatch()
-        }
-      case src: Array[Double] =>
-        dest match {
-          case dest: Array[Double] => impl(src.length, dest.length, (i, j) => dest(i) = src(j))
-          case _                   => mismatch()
-        }
-      case _ =>
-        mismatch()
-    })
+    } else
+      (src match {
```

Comment:
This is one of (many) examples where scalafmt moves things where we'd really need a block (i.e. braces).